py/torch_tensorrt/dynamo/debug/_Debugger.py

```diff
@@ -92,6 +98,26 @@ def __init__(
         self.capture_fx_graph_before = capture_fx_graph_before
         self.capture_fx_graph_after = capture_fx_graph_after
 
+        if self.cfg.capture_shim:
+            if not sys.platform.startswith("linux"):
+                _LOGGER.warning(
+                    "capture_shim featureis only supported on linux, will disable it"
```

Comment:
`Capturing TensorRT API calls is only supported on Linux, therefore ignoring the capture_tensorrt_api_recording setting`